packages/itwinui-react/src/core/utils/hooks/useUncontrolledState.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+
+/**
+ * Wrapper over `useState` that always gives preference to the
+ * uncontrolled state (which often comes from a prop).
+ */
+export const useUncontrolledState = <T>(uncontrolledState: T) => {
+  const [controlledState, setControlledState] =
+    React.useState(uncontrolledState);
```

Comment:
Do we need to initialize `controlledState` on L13? Because:
* If `uncontrolledState` is passed, then we don't use `controlledState` (L15), so no benefit of initializing it.
* If `uncontrolledState` is not passed, we are initializing controlledState with `undefined`

Reply:
ok, removed [`3cc41d6`](https://github.com/iTwin/iTwinUI/pull/1477/commits/3cc41d64cf8dbe5eae2f63eb9e651210b052d8eb)